From 02039a00c3043b5bfa5f498ae85423fc1d60ec63 Mon Sep 17 00:00:00 2001
From: Myron Stowe <myron.stowe@redhat.com>
Date: Thu, 26 Jun 2014 14:24:48 -0400
Subject: [Documentation] MAINTAINERS: Add Jingoo Han as Samsung Exynos PCIe driver maintainer

Message-id: <20140626142448.18681.64135.stgit@gir.stowe>
Patchwork-id: 84965
O-Subject: [RHEL7.1 PATCH 2/7] MAINTAINERS: Add Jingoo Han as Samsung Exynos PCIe driver maintainer
Bugzilla: 1110890
RH-Acked-by: Jes Sorensen <Jes.Sorensen@redhat.com>
RH-Acked-by: Dean Nelson <dnelson@redhat.com>
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>

Bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1110890
Upstream Status: 4af8225567065f1825baed259822b0d26c343fad

commit 4af8225567065f1825baed259822b0d26c343fad
Author: Jingoo Han <jg1.han@samsung.com>
Date:   Thu Sep 12 20:29:24 2013 +0900

    MAINTAINERS: Add Jingoo Han as Samsung Exynos PCIe driver maintainer

    Add myself as maintainer for Samsung Exynos PCIe driver.

    Signed-off-by: Jingoo Han <jg1.han@samsung.com>
    Signed-off-by: Bjorn Helgaas <bhelgaas@google.com>

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/MAINTAINERS b/MAINTAINERS
index bba2ab7..c8ad49f 100644
--- a/MAINTAINERS
+++ b/MAINTAINERS
@@ -6207,6 +6207,12 @@ S: Supported
 F: Documentation/devicetree/bindings/pci/nvidia,tegra20-pcie.txt
 F: drivers/pci/host/pci-tegra.c
 
+PCI DRIVER FOR SAMSUNG EXYNOS
+M: Jingoo Han <jg1.han@samsung.com>
+L: linux-pci@vger.kernel.org
+S: Maintained
+F: drivers/pci/host/pci-exynos.c
+
 PCMCIA SUBSYSTEM
 P: Linux PCMCIA Team
 L: linux-pcmcia@lists.infradead.org
-- 
1.7.1